### DISCUSSION TITLE
Possible to clear the ide output?

### DISCUSSION THREAD

**Discussion by pepeu93** [zero-plusplus/vscode-autohotkey-debug#145] [Q&A] [answered]
Hello, thank you for such an amazing extension.

I see that now it supports jump to the error line, would like to ask if it does also support clearing the vscode debug console?
I mean:
![image](https://user-images.githubusercontent.com/82878129/136644540-3ceaf385-1307-4730-a5a9-5790cc6c1362.png)

**Comment by zero-plusplus** [comment, marked as answer]
**I'm using machine translation.**

Thank you for using my extension.

You can see the following icon on the right side of the debug console, which is not visible in the your sample image. By clicking on it, you can clear the debug console.
![](https://raw.githubusercontent.com/microsoft/vscode-codicons/b2f3e2417164c213c6131c0f274588064b51e8dd/src/icons/clear-all.svg)

Alternatively, you can do the same thing by pressing `F1` and typing `workbench.debug.panel.action.clearReplAction` then enter.

**Reply by pepeu93** [reply]
@zero-plusplus 
I mean a way to clear the output by a command line, something similar to the debug breakpoint option:
`; @Debug-Breakpoint(x= 100)`
 
I'm debugging a code that runs when I call a hotkey, and each time i run it i need to clear the output because many lines there distract me. 
An option to automatically clear it would help.
Something like `;@Debug-ClearConsole`

**Reply by zero-plusplus** [reply]
I understand what you mean.

It is probably possible to implement it. I've opened a [new issue](https://github.com/zero-plusplus/vscode-autohotkey-debug/issues/147), so if you have any additional requests regarding this, please leave a comment there.

However, it may take some time as we are currently working on implementing a [major feature](https://github.com/zero-plusplus/vscode-autohotkey-debug/issues/88).
This work is expected to be finished within a month.

**Reply by zero-plusplus** [reply]
@pepeu93
The [Clear-Console directive](https://github.com/zero-plusplus/vscode-autohotkey-debug/wiki/ClearConsole-directive) has been implemented in 1.10.0.

Thanks for the great idea.
